portfolio/src/main/webapp/script.js

```diff
@@ -31,3 +31,12 @@ function addRandomFact() {
   const factContainer = document.getElementById('randomFact');
   factContainer.innerText = fact;
 }
+
+/**
+ * Adds a greeting when page loads.
+ */
+ function addGreeting(){
+  fetch('greeting').then(response => response.text()).then((greeting) => {
+    document.getElementById('greeting-container').innerText = greeting;
```

Comment:
Is there an element in your HTML file with the id greeting-containter? I couldn't find one, so this line will probably cause an error.

Reply:
The same happened to me here, I forgot to delete that function and add the TODO statement for getComments(). This function will be deleted.